domaintools/api.py

```diff
@@ -371,7 +371,7 @@ def iris_detect_monitors(self, include_counts=False, datetime_counts_since=None,
         return self._results('iris-detect-monitors', '/v1/iris-detect/monitors/', order=order, offset=offset,
                              limit=limit, items_path=('monitors',), response_path=(), **kwargs)
 
-    def iris_detect_new_domains(self, monitor_id=None, escalation_types=None, tlds=None, risk_score_ranges=None,
+    def iris_detect_new_domains(self, monitor_id=None, tlds=None, risk_score_ranges=None,
                                 mx_exists=None, discovered_since=None, changed_since=None, search=None, sort=None,
```

Comment:
Took out escalation_types from New Domains endpoint because this should never happen.